README.md

```diff
@@ -121,31 +103,31 @@ NSAttributedString *blueBirdString = blueBirds.attributedString;
 
 ## Concatenation
 
-You can concatenate an array of `BONText`s:
+You can concatenate an array of `BONChain`s:
 
 ```objc
-BONText *oneFish = BONChain.new.string(@"one fish").text;
-BONText *twoFish = BONChain.new.string(@"two fish").text;
-BONText *redFish = BONChain.new.string(@"red fish").textColor([UIColor redColor]).text;
-BONText *blueFish = BONChain.new.string(@"blue fish").textColor([UIColor blueColor]).text;
-BONText *separator = BONChain.new.string(@", ").text;
+BONChain *oneFish = BONChain.new.string(@"one fish");
+BONChain *twoFish = BONChain.new.string(@"two fish");
+BONChain *redFish = BONChain.new.string(@"red fish").textColor([UIColor redColor]);
+BONChain *blueFish = BONChain.new.string(@"blue fish").textColor([UIColor blueColor]);
+BONChain *separator = BONChain.new.string(@", ");
 
-NSAttributedString *string = [BONText joinTexts:@[ oneFish, twoFish, redFish, blueFish ] withSeparator:separator];
+NSAttributedString *string = [BONChain joinChains:@[ oneFish, twoFish, redFish, blueFish ] withSeparator:separator];
```

Comment:
Might be, might be. With the new appending syntax, which you could also use with `reduce()`, I'm not finding that I really need the `joinChains` method as much as I thought I would, so it may become deprecated in a future release.